src/test/resources/org/broadinstitute/hellbender/utils/SequenceDictionaryUtils/test.fasta.fai

```diff
@@ -2,3 +2,7 @@
 2	16000	16296	80	81
 3	16000	32544	80	81
 4	16000	48792	80	81
+chr5	101	65041	80	81
+chr6	101	65193	80	81
+chr7	404	65345	80	81
+chr8	202	65804	80	81
```

Comment:
Still have a strange mix of "chr1" and "1" style contig names in this sequence dictionary -- all of these contigs should follow the same naming scheme.